ebs_snapshots/snapshot_manager.py

```diff
@@ -49,7 +49,7 @@ def _create_snapshot(connection, volume, name=''):
     if not name:
         name = '{}-snapshot'.format(volume.id)
     connection.create_tags(
-        [snapshot.id], dict(Name=name, Creator='ebs-snapshots'))
+        [snapshot.id], dict(Name=name, creator='ebs-snapshots'))
```

Comment:
`Name` is something that is used by the AWS UI and tools and it's better to just keep it that way. I just treat it as not a tag, but a property we _have_ to set.